docs/apache-airflow-providers-docker/index.rst

```diff
@@ -49,7 +49,7 @@
     :maxdepth: 1
     :caption: Resources
 
-    Example DAGs <https://github.com/apache/airflow/tree/providers-docker/|version|/providers/tests/system/docker>
+    Example DAGs <https://github.com/apache/airflow/tree/providers-docker/|version|/tests/system/providers/docker>
```

Comment:
This seems to be wrong for all providers.
can you apply a fix for all?

Reply:
Done